src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -62,7 +62,7 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 			.sink { [weak self] _ in
 				guard let self = self else { return }
 				self.dynamicTableViewModel = self.viewModel.dynamicTableViewModel
-				DispatchQueue.main.asyncAfter(wallDeadline: .now() + 0.35) {
```

Comment:
I really wonder why this 0.35 was there in the first place. Seems quite specific.

Reply:
Checked already with Kai. In the past he discovered obscure timing issues but today testing didn't show any issues.

Reply:
maybe from some testing while development, I can't remember why

Reply:
Okay do we still need the Dispatch then?

Reply:
removed the obsolete dispatch

Reply:
Good point. Removed the obsolete dispatch.